projects/core/src/user/occ/user.service.spec.ts

```diff
@@ -256,4 +257,28 @@ describe('OccUserService', () => {
       mockReq.flush('');
     });
   });
+
+  describe('update password: ', () => {
+    it('should update the password for userId', () => {
+      const userId = 'test@test.com';
+      const oldPassword = 'OldPass123!';
+      const newPassword = 'NewPass456!';
+
+      service
+        .updatePassword(userId, oldPassword, newPassword)
+        .subscribe(result => expect(result).toEqual(''));
+
+      const mockReq = httpMock.expectOne(req => {
+        console.log(req);
+        console.log('ser body: ', req.serializeBody());
```

Comment:
Please remove `console.log`s from lines `272` and `273`

Reply:
Fixed.